### ISSUE TITLE
Use codesniffer without Generic and Squiz

### ISSUE THREAD

**Issue by duskohu** [squizlabs/PHP_CodeSniffer#3725]
**Describe the bug**
How can validate code only PSR2 standard, without Generic and Squiz?
I have config file, only for use tabs and remove Too Long Line Length.
Now I have in report `Squiz` and `Generic`

**Code sample**
```
phpcs --standard=.phpcs.xml
```

**Custom ruleset**
```xml
<?xml version="1.0"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="test">
    <file>./../../web</file>

    <arg name="standard" value="PSR2"/>
    <arg name="extensions" value="php"/>
    <arg name="basepath" value="./../../"/>
    <arg name="tab-width" value="4"/>

    <rule ref="PSR2">
        <exclude name="Generic.WhiteSpace.DisallowTabIndent"/>
        <exclude name="Generic.Files.LineLength.TooLong"/>
    </rule>

    <rule ref="Generic.WhiteSpace.DisallowSpaceIndent"/>

    <rule ref="Generic.WhiteSpace.ScopeIndent">
        <properties>
            <property name="indent" value="4"/>
            <property name="tabIndent" value="true"/>
        </properties>
    </rule>
</ruleset>
```

**Versions (please complete the following information):**
PHP_CodeSniffer version 3.7.1 (stable)

**Comment by jrfnl** [comment]
The PSR2 ruleset only applies the rules as they are defined in PSR2 and makes use of a variety of sniffs to do so.
So the PSR2 standard uses _sniffs_ from the Generic and Squiz standards, it doesn't use those standards as such, so you *are* validating only against PSR2.

**Comment by duskohu** [comment]
@jrfnlthx,  and how?

**Comment by jrfnl** [comment]
@duskohu Sorry, but I don't know what you are asking. If you want to check your code against PSR2, you will get sniffs from Generic and Squiz as otherwise, you are only checking code against a subset of rules from PSR2.
